app/src/main/res/raw/faq.markdown

```diff
@@ -8,7 +8,7 @@
 <a name="video-problems"></a>
 ### Die Videos laufen nicht – was soll ich tun?
 
-Wenn die Video-Streams auf Android nicht abspielen kann das mehrere Gründe haben:
+Wenn die Video-Streams auf Android nicht abspielen, kann das mehrere Gründe haben:
```

Comment:
Generally this is a bad German, just word for word translation from English. Better maybe: "Wenn die Video-Streams auf Android nicht funktionieren/abspielbar sind…"

Reply:
You're absolute right, I added your suggestion.